x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/job_details_step/components/additional_section/additional_section.tsx

```diff
@@ -25,21 +25,24 @@ export const AdditionalSection: FC<Props> = ({ additionalExpanded, setAdditional
         buttonContent={ButtonContent}
         onToggle={setAdditionalExpanded}
         initialIsOpen={additionalExpanded}
+        data-test-subj="mlJobWizardToggleAdditionalSettingsSection"
       >
-        <EuiSpacer />
+        <section data-test-subj="mlJobWizardAdditionalSettingsSection">
```

Comment:
curious why `section` is being used here, it seems to me to be a misuse of this tag.
A `div` might be more appropriate.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
so it's semantically a section for additional settings

Reply:
yes, I know what it is.
> Do not use the section element as a generic container; this is what div is for
IMO section is for documents, not web apps.

Reply:
It's not a problem, i was just wondering if there was some specific reason for using it here as i've only ever seen it being used for documents or webpages where the contents are similar in theme.
Searching through the rest of kibana i see It's been used in a similar way a couple of times.